src/App.tsx

```diff
@@ -46,13 +40,15 @@ export const App: React.FC = () => {
 
   return (
     <>
-      <Suspense fallback={<div className="fixed inset-0 bg-black"/>}>
+      {/* Background pattern appears immediately */}
+      <div className="fixed inset-0 z-[-2] h-screen w-screen bg-[#000000] bg-[radial-gradient(#ffffff33_1px,#00091d_1px)] bg-[size:20px_20px]"/>
+      {/* Shader overlay fades in after pattern */}
+      <Suspense fallback={null}>
```

Comment:
the fading in of the shader looks good, but there is still a noticable white flash before this background shows up. we need to improve that. the first thing I always see is the file drop component, which shows up instantly, which is good, but the background is extremely noticeable when it flashes from white to dark